tracks/r/exercises/raindrops.md

```diff
@@ -0,0 +1,60 @@
+### Possible solution
+
+This solution is very intuitive and follows step-by-step the instructions in the exercise
+
+```r
+raindrops <- function(number) {
+  rain <- ""
+
+  if (number %% 3 == 0)
+    rain <- append(rain, "Pling")
+  
+  if (number %% 5 == 0)
+    rain <- append(rain, "Plang")
+  
+  if (number %% 7 == 0)
+    rain <- append(rain, "Plong")
+  
+  if (!(number %% 3 == 0 | number %% 5 == 0 | number %% 7 == 0)) # alternatively: if(rain == "")
+    return(as.character(number))
+  
+  paste(rain, collapse = "")
+}
+
+```
+
+The learning goal here is (I guess) to avoid unnecessary conversion of inputs as strings and to account for inputs that are factors of more than one if-statement (e.g. 15).
+
+### Common suggestions
+
+- avoid a string transformation of the input at the beginning
+- go through the task iteratively to account for all possible factors
+- for-loops impact the readability here a bit, I think, so one could point learners to alternatives 
+
+### Talking points
+
+- more advanced learners might prefer to avoid several if conditions could be pointed to an alternative with an if-else statement:
```

Comment:
How about "a single" instead of "an"  here?

Reply:
good point!